zwave_js_server/exceptions.py

```diff
@@ -162,11 +163,32 @@ def __init__(self, value: "Value", path: str) -> None:
 class NotificationHasUnsupportedCommandClass(BaseZwaveJSServerError):
     """Exception raised when notification is received for an unsupported CC."""
 
-    def __init__(self, event: "Event", command_class: CommandClass) -> None:
+    def __init__(self, event: "Event", command_class: "CommandClass") -> None:
         """Initialize an invalid Command Class error."""
         self.event_data = event.data
         self.command_class = command_class
         super().__init__(
             "Notification received with unsupported command class "
             f"{command_class.name}: {event.data}"
         )
+
+
+class RssiErrorReceived(BaseZwaveJSServerError):
+    """Exception raised when an RSSI error is received."""
+
+    def __init__(self, error: "RssiError") -> None:
+        """Initialize an RSSI error."""
+        self.error = error
+        super().__init__()
+
+
+class RssiErrorReceivedInList(BaseZwaveJSServerError):
```

Comment:
Maybe rename this to make it easier for users to know which exception to catch?
```suggestion
class RepeaterRssiErrorReceived(BaseZwaveJSServerError):
```